PacificPower/readPP.js

```diff
@@ -662,6 +759,23 @@ let pp_meter_id = "";
   for (let i = 0; i < otherErrorArray.length; i++) {
     console.log(otherErrorArray[i]);
   }
+  console.log("\nMeters Excluded: ");
+  for (let i = 0; i < pp_meters_exclude.length; i++) {
+    console.log(pp_meters_exclude[i]);
+  }
+  console.log("\nMeters Included in DB: ");
+  for (let i = 0; i < pp_meters_include.length; i++) {
+    console.log(pp_meters_include[i]);
+  }
+  console.log("\nMeters Not Found in Exclusion List (new meters): ");
```

Comment:
Would it be a good idea to do a length check before logging the "Meters Not Found in Exclusion List (new meters):" text?

E.g. https://github.com/OSU-Sustainability-Office/automated-jobs/blob/main/check-acq/check-acq.js#L845

That way if there happen to be no new meters, nothing is logged, which makes detection easier on the AWS Cloudwatch side (I can also change this on the missing Data PR).

Reply:
LGTM